src/provision/azure/tasks/set-key-vault-secrets.yaml

```diff
@@ -0,0 +1,37 @@
+# Task to adds/updates variables in an Azure Key Vault
+parameters:
+  - name: keyVaultName # The name of the key vault
+    type: string
+    default: ''
```

Comment:
suggestion: remove the default value for `keyVaultName` and `serviceConnectionName`, as we'd always want the dev to specify this

Reply:
Good point, cheers